packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.base.tsx

```diff
@@ -108,9 +116,7 @@ export class SwatchColorPickerBase extends React.Component<ISwatchColorPickerPro
     return (
       <Grid
         {...this.props}
-        items={colorCells.map((item, index) => {
-          return { ...item, index: index };
-        })}
+        items={this._getItemsWithIndex(colorCells)}
```

Comment:
Can you add a note on the `colorCells` prop documentation warning that consumers should avoid mutating the array to prevent unnecessary re-renders (or something along those lines)?

Reply:
Sure.

Reply:
Done added note explaining why this will prop could be a little dangerous to change.